markdown/sync.md

```diff
@@ -37,9 +53,9 @@ nf-core create --no-git -n 'YOURPIPELINENAME' -d 'YOUR ACTUAL PIPELINE DESCRIPTI
 This creates a new directory `YOURPIPELINENAME` with the template pipeline files in it.
 Now move these files into your root git directory:
 ```bash
-mv YOURPIPELINENAME/* .
-mv YOURPIPELINENAME/.[!.]* .
-rm -rf YOURPIPELINENAME
+mv nf-core-YOURPIPELINENAME/* .
```

Comment:
The `nf-core create` command now has a `-o`/`--outdir` option so this isn't required any more. You can instead just run:

```
 nf-core create --no-git --name 'YOURPIPELINENAME' --description 'YOUR ACTUAL PIPELINE DESCRIPTION' --outdir ./
```